Loop/Managers/LoopDataManager.swift

```diff
@@ -71,7 +71,8 @@ final class LoopDataManager {
             defaultAbsorptionTimes: LoopSettings.defaultCarbAbsorptionTimes,
             carbRatioSchedule: carbRatioSchedule,
             insulinSensitivitySchedule: insulinSensitivitySchedule,
-            overrideHistory: overrideHistory
+            overrideHistory: overrideHistory,
+            carbAbsorptionModel: FeatureFlags.nonlinearCarbModelEnabled ? .nonlinear : .linear
```

Comment:
No, the user (as in a Tidepool Loop user) cannot change this value.  FeatureFlags are compile time.

Reply:
Should the user be able to change this value?

Reply:
No. Tidepool 1.0 will be using the linear model, because the Jaeb study data is primarily using that. We (Tidepool) need to decide *if* this will be included in a subsequent update, and how. DIY Loop does not allow the user to change this via UI, although users could change in the code when they build their app.